orchestra/communication/slack.py

```diff
@@ -95,6 +95,21 @@ def create_project_slack_group(project):
     return project.slack_group_id
 
 
+@run_if('ORCHESTRA_SLACK_EXPERTS_ENABLED')
+def archive_project_slack_group(project):
+    """
+    Archive a slack channel of a project
+    """
+    slack = OrchestraSlackService()
+    try:
+        response = slack.groups.archive(project.slack_group_id)
+        is_archived = response.body['ok']
+        if not is_archived:
+            logger.exception(response.body['error'])
+    except SlackError:
```

Comment:
this is nice! can you replace line 68-71 with this same exact `except`/`logger.exception` sequence?